test/functional/Java8andUp/playlist.xml

```diff
@@ -2672,5 +2672,27 @@
 			<impl>ibm</impl>
 		</impls>
 	</test>
+	
+	<test>
+		<testCaseName>DupClassNameTest</testCaseName>
+		<command>$(ADD_JVM_LIB_DIR_TO_LIBPATH) $(JAVA_COMMAND) $(JVM_OPTIONS) -Xshareclasses:name=DupClassNameTest,reset \
```

Comment:
Do we need `ADD_JVM_LIB_DIR_TO_LIBPATH `?

Reply:
I removed `ADD_JVM_LIB_DIR_TO_LIBPATH` .